[2024-05-20T05:29:36.973+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-20T05:29:36.991+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL.transformar_data manual__2024-05-20T05:29:35.354455+00:00 [queued]>
[2024-05-20T05:29:36.997+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL.transformar_data manual__2024-05-20T05:29:35.354455+00:00 [queued]>
[2024-05-20T05:29:36.998+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-20T05:29:37.008+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar_data> on 2024-05-20 05:29:35.354455+00:00
[2024-05-20T05:29:37.012+0000] {standard_task_runner.py:63} INFO - Started process 77 to run task
[2024-05-20T05:29:37.015+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL', 'transformar_data', 'manual__2024-05-20T05:29:35.354455+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpkpf_tit2']
[2024-05-20T05:29:37.015+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask transformar_data
[2024-05-20T05:29:37.046+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL.transformar_data manual__2024-05-20T05:29:35.354455+00:00 [running]> on host 8251e45c28eb
[2024-05-20T05:29:37.103+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_ETL' AIRFLOW_CTX_TASK_ID='transformar_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-20T05:29:35.354455+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-20T05:29:35.354455+00:00'
[2024-05-20T05:29:37.103+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-20T05:29:37.563+0000] {logging_mixin.py:188} INFO - {'Error Message': 'Invalid API KEY. Feel free to create a Free API Key or visit https://site.financialmodelingprep.com/faqs?search=why-is-my-api-key-invalid for more information.'}
[2024-05-20T05:29:37.564+0000] {python.py:237} INFO - Done. Returned value was: {'Error Message': 'Invalid API KEY. Feel free to create a Free API Key or visit https://site.financialmodelingprep.com/faqs?search=why-is-my-api-key-invalid for more information.'}
[2024-05-20T05:29:37.565+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-20T05:29:37.581+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ETL, task_id=transformar_data, run_id=manual__2024-05-20T05:29:35.354455+00:00, execution_date=20240520T052935, start_date=20240520T052936, end_date=20240520T052937
[2024-05-20T05:29:37.628+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-20T05:29:37.648+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-20T05:29:37.650+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
